src/main/java/net/starschema/clouddb/jdbc/BQForwardOnlyResultSet.java

```diff
@@ -593,12 +598,13 @@ public void deleteRow() throws SQLException {
    */
   @Override
   public int findColumn(String columnLabel) throws SQLException {
-    if (this.isClosed()) {
+    if (isClosed()) {
       throw new BQSQLException("This Resultset is Closed");
     }
-    int columncount = this.getMetaData().getColumnCount();
-    for (int i = 1; i <= columncount; i++) {
-      if (this.getMetaData().getCatalogName(i).equals(columnLabel)) {
+    final ResultSetMetaData metadata = this.getMetaData();
```

Comment:
Is it worth extracting this duplicate `for` loop into a helper that can be called by both `BQForwardOnlyResultSet` and `BQResultSet`? Up to you

Reply:
Realize its dup'd some more below, maybe worth extracting